python/analysis/auto_generated/mesh/qgsmeshcalculator.sip.in

```diff
@@ -46,6 +45,8 @@ implemented and supported
     };
 
     QgsMeshCalculator( const QString &formulaString,
+                       const QString &outputDriver,
```

Comment:
This is an API break, right?

Reply:
I could teoretically encode it all to previous "outputFile" parameter is some form list DRIVER:path:groupname or so, but not sure if it is worth to strictly keep the mesh API (where it has all over the place notice it is EXPERIMENTAL)...

Reply:
added that api back with deprecated flag